tools/flang2/utils/upper/upperilm.in

```diff
@@ -674,6 +732,10 @@ Q2K	ilm
 QTOI    ilm     ilm
 QTOK    ilm     ilm
 QTOQ    ilm     ilm
+QATAN   ilm
```

Comment:
similar code above is hidden behind the #ifdef. Why not here?

Reply:
The #ifdef had already been added on line 706 by commit 46ec115b.